schema/filter_artworks.js

```diff
@@ -80,6 +80,9 @@ const FilterArtworks = {
     major_periods: {
       type: new GraphQLList(GraphQLString),
     },
+    partner_id: {
+      type: GraphQLString,
```

Comment:
if i use the other endpoint, this change is unnecessary. however, the api supports it, so should i just leave it in? could be useful for actual filtered artwork search if we were hitting metaphysics instead of gravity.